pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -72,24 +72,29 @@ buildDotnetModule rec {
 
   doCheck = true;
 
-  dotnetTestFlags = [
-    "--environment=USER=nobody"
-    (
-      "--filter="
-      + lib.strings.concatStringsSep "&" (
+  dotnetTestFlags =
+    let
+      # NOTE: Can't use `disabledTests` because we have additional category based filters
+      filters =
         [
           "Category!=Disabled"
           "FlakeyTest!=True"
           "RequiresNetworking!=True"
-          "FullyQualifiedName!=NexusMods.UI.Tests.ImageCacheTests.Test_LoadAndCache_RemoteImage"
-          "FullyQualifiedName!=NexusMods.UI.Tests.ImageCacheTests.Test_LoadAndCache_ImageStoredFile"
         ]
-        ++ lib.optionals (!enableUnfree) [
-          "FullyQualifiedName!=NexusMods.Games.FOMOD.Tests.FomodXmlInstallerTests.InstallsFilesSimple_UsingRar"
-        ]
-      )
-    )
-  ];
+        ++ lib.map (n: "FullyQualifiedName!=${n}") (
```

Comment:
This seems like premature abstraction (as soon as something other than `FullyQualifiedName!=${n}` is necessary for `!enableUnfree` this would have to be unpicked), but feel free to leave it in if you prefer.

Reply:
This is basically just re-implementing `disabledTests` from buildDotnetModule but with support for having additional filters.

I think it's unlikely we'll need other filters optionally, but if we do we can either have multiple conditions or deal with it some other way at the time.

The best (long-term)
solution is that buildDotnetModule adds support for additional `testFilters`, so we don't have to build the `--filter` flag ourselves.

Reply:
I think we're talking about different things, but feel free to leave it in.

Reply:
I've dropped these changes